app/src/main/java/io/gnosis/safe/ui/transaction/TransactionView.kt

```diff
@@ -7,7 +7,7 @@ import androidx.annotation.StringRes
 import io.gnosis.data.models.TransactionStatus
 import pm.gnosis.model.Solidity
 
-sealed class TransactionView(open val status: TransactionStatus) {
+sealed class TransactionView(open val status: TransactionStatus?) {
```

Comment:
Maybe instead of making this nullable, we should leave it as is and add `TransactionStatus.Unknown`. Then we don't need to program so defensively. As far as I can see, this is null so you can pass `null` to `object Unknow : TransactionView(null)` ?

Reply:
it is overridden in the subclasses and it is not null there

Reply:
Adding Unknown status would mean we have to extend all of when statements to account for it. It is just more boilerplate code

Reply:
Ok sounds good. Then let's not do it.